bedtools_genomecov_v2.28.0_cv2/bedtools_genomecov.cwl

```diff
@@ -0,0 +1,81 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+id: bedtools_genomecov
+baseCommand:
+  - bedtools
+  - genomecov
+inputs:
+  - id: input
+    type: File
+    inputBinding:
+      position: 5
+      prefix: '-ibam'
+      shellQuote: false
+    doc: >-
+      The input file can be in BAM format (Note: BAM  must be sorted by
+      position)
+    secondaryFiles:
+      - ^.bai
+  - id: output_file_name
+    type: string
+  - id: memory_overhead
+    type: int?
+  - id: memory_per_job
+    type: int?
+  - id: number_of_threads
+    type: int?
+outputs:
+  - id: output_file
+    type: File
+    outputBinding:
+      glob: |2-
+          ${
+            if (inputs.output_file_name.length>0)
+              return inputs.output_file_name;
+            return inputs.input.basename.replace('.bam', '.bedgraph');
+          }
+label: bedtools_genomecov
+arguments:
+  - position: 0
+    prefix: ''
+    separate: false
+    valueFrom: '-bg'
```

Comment:
I just realized the -bg should be regular input parameter with boolean any specific reason this is in arguments section vs input section

Reply:
done